letsencrypt/cli.py

```diff
@@ -383,6 +383,12 @@ def _suggest_donate():
     reporter_util.add_message(msg, reporter_util.LOW_PRIORITY)
 
 
+def _report_successful_dry_run():
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+    reporter_util.add_message("The dry run was successful.",
```

Comment:
Maybe elaborate a little here?  "Using these options, we appear to be able to pass ACME Challenges and obtain a cert for <names>".